jena-cmds/src/main/java/tdb2/tdbcompact.java

```diff
@@ -18,18 +18,36 @@
 
 package tdb2;
 
+import org.apache.jena.base.Sys;
+import org.apache.jena.cmd.ArgDecl;
 import org.apache.jena.tdb2.store.DatasetGraphSwitchable;
 import org.apache.jena.tdb2.sys.DatabaseOps;
 import tdb2.cmdline.CmdTDB;
 
 public class tdbcompact extends CmdTDB {
+    private static final ArgDecl argDeleteOld = new ArgDecl(ArgDecl.NoValue, "deleteOld");
```

Comment:
This calls it `--deleteOld` but the help is `--delete`.

Reply:
whoops. good catch.
fixed.